src/App.js

```diff
@@ -1,14 +1,10 @@
 import React from 'react';
-import Calculator from './components/calculator';
+import { Calculator } from './components/calculator';
 
-// eslint-disable-next-line react/prefer-stateless-function
-export default class App extends React.Component {
-  render() {
-    return (
-      <div className="container">
-        <h1 className="row justify-content-center">Math Magicians</h1>
-        <Calculator />
-      </div>
-    );
-  }
-}
+/* eslint-disable react/prefer-stateless-function,  import/prefer-default-export */
```

Comment:
- [ ] Let's remove these disabled rules, they are not necessary anymore and we can refactor some code if necessary. Remember that the purpose of linters is to check that our code is clean and standardized.